dev/Vagrantfile

```diff
@@ -104,6 +105,16 @@ Vagrant.configure("2") do |config|
     attributes['vm']['elasticsearch'] = nil
   end
 
+  if attributes['vm'].has_key? 'external_solr'
+    if attributes['vm']['external_solr']['start'] == true
```

Comment:
It totally makes sense...  but for whatever reason all the other attributes['vm'] clauses use this strange pattern.  It might just be copy-pasta, but I it'd be better to just fix them all up in another pass.  Some of this (the chef_backend stuff assigning nil but then reading an attribute underneath it later) doesn't even seem quite right but those codepaths may never ever get used.  I might simplify it a bunch.